RedHatInsights/vulnerability-ui
# POC: CVE detail page hybrid tabs

Resolves: [RHIF-296](https://issues.redhat.com/browse/RHIF-296)

Implements hybrid inventory tabs on the CVE details page. 

To test:

1. Run the PR locally with insights-inventory app and log in using the insights-qa account
2. Navigate to the CVEs page and find `CVE-2023-5197`
3. Click on the cve to get into the details page
4. Observe that now there are 2 tabs with the first one showing conventional systems and the next immutable devices
5. Observe following
           System name links to inventory -> system details with vuln tab in focus
          Advisory links to advisory on access.com if there is an advisory available
          Image links to Images -> Immutable Image ->Image set Details 
6. Verify following filters working: Name, Advisory, OS, Group
7. Table is sorted by enabled columns
